src/app/reviews/reviews.component.html

```diff
@@ -0,0 +1,53 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-title center>Reviews</ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content padding color="light">
+  <div class="top-round-btns" text-center>
+    <ion-button class="do" [ngClass]="{'active-review': !showDo}" (click)="!showDo && activeDo()">Todo</ion-button>
+    <ion-button class="done" [ngClass]="{'active-review': !showDone}" (click)="!showDone && activeDone()">Done</ion-button>
+  </div>
+  <div *ngIf="!reviews.length; else reviewsCard" class="no-review-show" padding text-center>
+    <div class="empty-status" >
+      <h2 text-center>You have no new submissions to review</h2>
+    </div>
+  </div>
+  <ng-template #reviewsCard>
+    <ng-container *ngFor="let review of reviews">
+      <ion-card
+        color="primary"
+        class="card practera-card"
+        *ngIf="!review.isDone && !showDo" 
+        (click)="gotoReview(review.id)"
+      >
+        <ion-card-content class="review">
+          <h4 color="primary-contrast" class="activity-name">{{review.name}}</h4>
```

Comment:
`color=""` these kind of directives only works for ionic component, they don't work for normal html tags.
Also, the color of element that has a background color will use the `*-contrast` color. That means you don't need to specify it. Same for elements below